easybuild/easyblocks/p/psmpi.py

```diff
@@ -29,12 +29,60 @@
 """
 
 import easybuild.tools.toolchain as toolchain
+import os
+import re
 
 from distutils.version import LooseVersion
 from easybuild.easyblocks.mpich import EB_MPICH
 from easybuild.framework.easyconfig import CUSTOM
-from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.build_log import EasyBuildError, print_msg
+from easybuild.tools.filetools import remove_file, write_file
 from easybuild.tools.modules import get_software_root
+from easybuild.tools.run import run_cmd
+
+
+PINGPONG_PGO_TEST = """
+/****************************************
+ * Buggy, ugly and extremely simple MPI ping pong.
+ * The goal is to trigger pscom internal routines
+ * to generate profiles to be used by PGO-enabled
+ * compilers.
+ *
+ * Nothing more, nothing less.
+ *
+ * We try small and large messages to walk the path for
+ * eager and rendezvous protocols
```

Comment:
@damianam Where did this come from? Should mention author or upstream source?

Reply:
There is no upstream source or author other than me. It is a very simple test. I based it on a hello world example, but I don't think it should matter too much......